interfaces/system_key_test.go

```diff
@@ -48,10 +49,17 @@ func (s *systemKeySuite) SetUpTest(c *C) {
 	id, err := osutil.MyBuildID()
 	c.Assert(err, IsNil)
 	s.buildID = id
+
+	s.restorers = []func(){
+		osutil.MockMountInfo(""), osutil.MockEtcFstab(""),
```

Comment:
Thinking about this, maybe some mocks here that are non-empty?

Reply:
What would the purpose of those be?